src/hdf5.h

```diff
@@ -41,8 +41,20 @@
 #include "H5VLpublic.h" /* Virtual Object Layer                     */
 #include "H5Zpublic.h"  /* Data filters                             */
 
-/* "developer-oriented" headers (for VFD, VOL, etc. authors) */
-#include "hdf5dev.h"
+/* Application developer headers for various interfaces */
```

Comment:
These are for plugin authors - definitely not for application developers. :-)

Reply:
I copied that out of the hdf5dev.h header that you authored :)

Will change, though.

Reply:
Thanks - dunno where I was at with that, or where I pasted it from... :-)